storage/norcow_blockwise.h

```diff
@@ -72,11 +72,10 @@ static secbool write_item(uint8_t sector, uint32_t offset, uint16_t key,
 
     *pos = FLASH_ALIGN(offset + NORCOW_VALID_FLAG_LEN + len);
     if (data != NULL) {
-      // write key and first data part
+      // write flag and first data part
       uint16_t len_to_write =
           len > NORCOW_DATA_OPT_SIZE ? NORCOW_DATA_OPT_SIZE : len;
-      memset(block, 0, sizeof(block));
-      block[0] = NORCOW_VALID_FLAG;
```

Comment:
We should write the valid flag always. If the definition of `NORCOW_VALID_FLAG` changes or we start `memset`ing to something else, then the code will break.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/de177fd1aa86751e05d2fae01e647f4937054315